Cargo.toml

```diff
@@ -13,7 +13,15 @@ Timeouts for futures.
 """
 
 [dependencies]
+gloo-timers = { version = "0.2.0", features = ["futures"], optional = true }
+send_wrapper = { version = "0.4.0", optional = true }
 
 [dev-dependencies]
 async-std = { version = "1.0.1", features = ["attributes"] }
 futures = "0.3.1"
+
+[features]
+wasm-bindgen = [
+	"gloo-timers",
+	"send_wrapper"
```

Comment:
This is indentation of 6; indentation of 4 would suffice.

Reply:
Those were tabs, but I've switched them to spaces.